capstone-project/src/main/angular-webapp/src/app/authentication/authentication.component.ts

```diff
@@ -34,4 +39,8 @@ export class AuthenticationComponent implements OnInit {
     return this.userService.getUser();
   }
 
+  // Return the userService
+  getUserService() {
+    return this.userService;
+  }
```

Comment:
It's bad practice to add methods just for testing. 

Can you avoid using that?